site/content/features/textblocks.md

```diff
@@ -0,0 +1,444 @@
+---
+title: Text Blocks (JEP 378)
+copyright: Cay S. Horstmann 2020. All rights reserved.
+jep: 378
+jdkversion: 15
+type: "sandbox"
+---
+
+
+Text blocks, added in their final form to Java 15, are a new form of string literals that is particularly suited for multiline text. Their usage is mostly straightforward, but you need to be aware how whitespace and escape sequences are processed.
+
+## The String Literals Landscape
+
+Sometimes, you need string literals that contain code in another programming language, such as HTML or SQL code, or perhaps plain text such as a license statement. With the classic Java string literals, that's not a lot of fun:
+
+```
+String myDoc = "<html>\n  <head>\n    <title>My Document</title>\n  </head>"
+  + "  <body>\n"
+  + "    <p>Hello, World!</p>\n"
+  + "  </body>\n</html>";
+```
+
+Other programming languages have string literal types that allow newlines. For example, in JavaScript, you can use a template literal:
+
+```
+let myDoc = `<html>
+  <head>
+    <title>My Document</title>
+  </head>
+  <body>
+    <p>Hello, World!</p>"
```

Comment:
Is the single quote sign intentional at this position? If usage of quotes should be demonstrated what about this?

    <p>Hello, "World"!</p>

Reply:
It was a typo and I will remove it. I don't think it's worth quoting the "World". It would detract from the newlines.